packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/drawNode.css

```diff
@@ -41,15 +43,23 @@ limitations under the License.
   color: #fff;
 }
 
+.DiffNode.is-removed {
+  background: #cc1616;
+  border: none;
+  color: #fff;
+}
+
 .DiffNode.is-less {
   border-color: #cc1616;
   background: #ffa39e;
 }
 
-.DiffNode.is-removed {
-  background: #cc1616;
-  border: none;
-  color: #fff;
+.TraceDiffGraph--dag.is-small .DiffNode--body {
+  opacity: 0;
+}
+
+.DiffNode--popover .DiffNode.is-ui-find-match {
+  outline: #fff3d7 solid 3px;
```

Comment:
Unfortunately not, as the `.DiffNode` within the `.DiffNode--popover` cannot inherit the rest of the outline specifications as its parent does not have an outline. I could either tell a couple parts of `antd` to inherit the transparent `outline`, or have a slightly redundant `outline` definition here. I chose this because it decouples our styling from `antd`'s html structure.

Reply:
Will resolve as we discussed in person.